debug.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Controls debugging behavior {@default false}
+ *
+ * - `true` : all debug options are enabled
+ * - `false` : all debug options are disabled
+ * - DebugOptions:
+ *   - `query` : Log queries
+ *   - `notices` : Log notices
```

Comment:
```suggestion
 * Controls debugging behavior. If set to `true`, all debug options are enabled.
 * If set to `false`, all debug options are disabled.
 *
 * {@default false}
```
I don't think we need to repeat information for `DebugOptions`.